documentation/guide-alternate-microservice.asciidoc

```diff
@@ -0,0 +1,722 @@
+:toc: macro
+toc::[]
+
+:doctype: book
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
+== Microservices in devonfw
+
+=== Introduction
+
+The Microservices architecture is an approach for application development based on a series of _small_ services grouped under a business domain. Each individual service runs autonomously and communicating with each other through their APIs. That independence between the different services allows to manage (upgrade, fix, deploy, etc.) each one without affecting the rest of the system's services. In addition to that the microservices architecture allows to scale specific services when facing an increment of the requests, so the applications based on microservices are more flexible and stable, and can be adapted quickly to demand changes.
```

Comment:
This is a very broad introduction and also the filename is very general.
My suggestion would be:
* rename the file to `alternative-microservice-netflix.asciidoc` (for naming conventions also see https://github.com/devonfw/docgen/blob/master/documentation/guidelines.asciidoc)
* create a `guide-microservice.ascidoc` with this general introduction (if we do not even have it or want in for `devonfw` in general somewhere in `devonfw-guide`). From there link to related options such as `netflix` and this alternative but also to `service-client`, etc.
* Remove this introduction here and give it a specific title such as `Microservices based on Netflix-Tools`.